src/steps/update-signatures/builders.ts

```diff
@@ -31,6 +31,26 @@ function builderConvertTsTypeToKeyword(tsType: string) {
   }
 }
 
+export function builderCreateArgsNode(signature: Signature) {
+  const members: unknown[] = [];
+
+  (signature.Args ?? []).forEach((argumentName) => {
```

Comment:
**TODO**

Currently, `signature.Args` can be `undefined` if the template file is not present. I don't think this is a valid case to consider.

Refactor code so that `signature.Args` is always of type `string[]`.